### PR TITLE
Update react-native-lightbox to react-native-lightbox-v2

### PR DESCRIPTION
**Pull request by Eloi-Perez**
Changed to an updated version of Lightbox with corrected errors.

### CODE REVIEW

**Review by alexjou — APPROVED** [review]

**Comment by theneekz** [comment]
Please update package.json to use the new package too and then merge this!

**Comment by Johan-dutoit** [comment]
If you're able to resolve the conflicts, then this can be merged in.

File: package.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     "prop-types": "15.7.2",
     "react-native-communications": "2.2.1",
     "react-native-iphone-x-helper": "1.3.1",
-    "react-native-lightbox": "0.8.1",
+    "react-native-lightbox-v2": "^0.8.8",
```
Comment (Johan-dutoit) [inline package.json:75]: 0.9.0 exists

File: src/MessageImage.tsx [hunk 2]
```diff
@@ -10,7 +10,8 @@ import {
   ImageStyle,
 } from 'react-native'
 // TODO: support web
-import Lightbox from 'react-native-lightbox'
+// @ts-ignore
```
Comment (Johan-dutoit) [inline src/MessageImage.tsx:13]: No need to ignore this, as I see it's built in typescript.

**Review by Johan-dutoit — CHANGES_REQUESTED** [review]
Also worth updating the example to use v2 as well.
Thanks!

**Comment by Johan-dutoit** [comment]
I've made the changes locally.  No need to worry about the patch.
Thanks for the contribution!

**Pull request closed without merging**